app/upgrades/v20/upgrades.go

```diff
@@ -23,6 +24,9 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		// Initialize the newly created param
+		keepers.ConcentratedLiquidityKeeper.SetParam(ctx, cltypes.KeyUnrestrictedPoolCreatorWhitelist, []string{})
```

Comment:
This feels like something that the upstream SDK should automate. WDYT about opening up an issue?

Reply:
I will open up an issue rn and link it here, good idea

Reply:
Created https://github.com/cosmos/cosmos-sdk/issues/17826#issue-1905750219